common/app/assets/assets.scala

```diff
@@ -152,6 +152,7 @@ class Assets(base: String, assetMapPath: String = "assets/assets.map") extends L
 
      val curl: String = RelativePathEscaper.escapeLeadingDotPaths(inlineJs("assets/curl-domReady.js"))
      val omnitureJs: String = inlineJs("assets/vendor/omniture.js")
+     val reactPath: String = "https://cdnjs.cloudflare.com/ajax/libs/react/0.13.2/react.min.js"
```

Comment:
Agreed. I looked into having a fallback so we could have the best of both worlds, but that's not possible with curl’s API.

I'll raise a subsequent PR for this. Added to the to dos.